tests/Rules/UnusedPublicPropertyRule/UnusedPublicPropertyRuleTest.php

```diff
@@ -51,6 +52,16 @@ public static function provideData(): Iterator
             [[$errorMessage, 7, RuleTips::SOLUTION_MESSAGE]],
         ];
 
+        $errorMessage = sprintf(
+            UnusedPublicPropertyRule::ERROR_MESSAGE,
+            LocallyUsedStaticPropertyViaStatic::class,
+            'somePublicStaticProperty'
+        );
+        yield [
+            [__DIR__ . '/Fixture/LocallyUsedStaticPropertyViaStatic.php'],
```

Comment:
btw: I wondered why we separate classes between `Fixture` and `Source` in this repo.

it might make sense todo so in a rector package, but I think its not useful in a phpstan rule/extension package?

Reply:
You're right, it can be only `/Fixture` here, if a file is provided in list of files analysed by PHPStan.